src/components/DuosDatePicker.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import {createTheme, ThemeProvider} from '@mui/material/styles';
+import {
+  DesktopDatePicker,
+  LocalizationProvider,
+  PickersActionBarProps,
+  PickersDay,
+  PickersDayProps
+} from '@mui/x-date-pickers';
+import {AdapterDayjs} from '@mui/x-date-pickers/AdapterDayjs';
+import {Button} from '@mui/material';
+import DialogActions from '@mui/material/DialogActions';
+import {Dayjs} from 'dayjs';
+import type {} from '@mui/x-date-pickers/themeAugmentation';
```

Comment:
It is interesting that intellij thinks this is unused.

![Screenshot 2024-12-10 at 2 33 14 PM](https://github.com/user-attachments/assets/713e1c42-804c-4bbd-94fe-502aa2dbc872)

Reply:
It is, and it definitely breaks when the import is removed.  :-)